hls4ml/backends/catapult/passes/convolution_templates.py

```diff
@@ -94,6 +94,9 @@ def format(self, node):
         else:
             params['fill_fn'] = 'FillConv1DBuffer'
 
+        params.setdefault('min_width', node.get_attr('in_width'))
```

Comment:
I think these can be replaced with just `params['min_witdth'] = node.get_attr('in_width')` and similar. Note that params is created on line 82, so `params['min_witdth']` is not set. The setdefaults has the same effect as `params['min_witdth'] = node.get_attr('in_width')`, so it's better to just use the more straightforward form.